elasticdl/python/tests/data_reader_test.py

```diff
@@ -58,6 +58,31 @@ def test_recordio_data_reader(self):
                     self.assertEqual(len(v.numpy()), 1)
 
 
+class CSVDataReaderTest(unittest.TestCase):
+    def test_csv_data_reader(self):
+        with tempfile.TemporaryDirectory() as temp_dir_name:
+            num_records = 128
+            columns = [
+                "sepal_length",
+                "sepal_width",
+                "petal_length",
+                "petal_width",
+                "class",
+            ]
+            iris_file_name = create_iris_csv_file(
+                size=num_records, columns=columns, temp_dir=temp_dir_name
+            )
+            csv_data_reader = CSVDataReader(columns=columns)
+            task = _MockedTask(0, num_records, iris_file_name)
+            records_count = 0
+            last_record = None
+            for record in csv_data_reader.read_records(task):
+                records_count += 1
+                last_record = record
+            self.assertEquals(records_count, num_records)
+            self.assertEquals(len(last_record), 5)
+
```

Comment:
It's better to add a unit test that generates a `tf.data.Dataset` instance with CSVDataReader.